api/BaseAudioContext.json

```diff
@@ -699,7 +699,7 @@
               "webview_android": "mirror"
             },
             "status": {
-              "experimental": false,
```

Comment:
Supported in Firefox https://searchfox.org/mozilla-central/source/dom/webidl/PeriodicWave.webidl#14